### PR TITLE
Add --always

### PR DESCRIPTION
**Pull request by wtanaka**

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/gradle-git-version](https://github.com/palantir/gradle-git-version), @wtanaka! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

**Comment by wtanaka** [comment]
@uschi2000 @markelliot

File: src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy [hunk 1]
```diff
@@ -109,11 +110,13 @@ class GitVersionPluginTests extends Specification {
         git.add().addFilepattern('.').call()
         git.commit().setMessage('initial commit').call()
 
+        String expected = git.getRepository().getRef("HEAD").getObjectId().abbreviate(VERSION_ABBR_LENGTH).name()
```
Comment (uschi2000) [inline src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy]: the test method name is outdated.

File: src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy [hunk 2]
```diff
@@ -129,12 +132,13 @@ class GitVersionPluginTests extends Specification {
         git.add().addFilepattern('.').call()
         git.commit().setMessage('initial commit').call()
         dirtyContentFile << 'dirty-content'
+        String expected = git.getRepository().getRef("HEAD").getObjectId().abbreviate(VERSION_ABBR_LENGTH).name()
```
Comment (uschi2000) [inline src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy]: can you turn this into a method, and also keep the VERSION_ABBR_LENGTH variable local to that method?

**Review by uschi2000 — CHANGES_REQUESTED** [review]
I'm fine with this, modulo two nitpicks.

File: src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy (hunk 1 above)
Comment (wtanaka) [reply src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy]: updated

File: src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy (hunk 2 above)
Comment (wtanaka) [reply src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy]: done

**Comment by wtanaka** [comment]
@uschi2000 updated

**Review by uschi2000 — APPROVED** [review]

**Pull request merged**